src/ImageSharp.Drawing/Processors/DrawPathProcessor.cs

```diff
@@ -26,65 +25,48 @@ public class DrawPathProcessor<TColor> : ImageProcessor<TColor>
         private const float AntialiasFactor = 1f;
         private const int PaddingFactor = 1; // needs to been the same or greater than AntialiasFactor
 
-        private readonly IPen<TColor> pen;
-        private readonly IPath[] paths;
-        private readonly RectangleF region;
-        private readonly GraphicsOptions options;
-
         /// <summary>
         /// Initializes a new instance of the <see cref="DrawPathProcessor{TColor}" /> class.
         /// </summary>
         /// <param name="pen">The pen.</param>
-        /// <param name="shape">The shape.</param>
+        /// <param name="region">The region.</param>
```

Comment:
Let's try to be a little more descriptive with params. e.g. The region of interest, The region withing the image to draw, etc. Remember... You're the only person who really understands this stuff 😝